pkg/sql/schemaexpr/check_constraint_test.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package schemaexpr
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/parser"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/builtins"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/sql/types"
+)
+
+func TestCheckConstraintBuilder_MarkNameInUse(t *testing.T) {
+	// TODO(mgartner);
+}
+
+func TestCheckConstraintBuilder_Build(t *testing.T) {
```

Comment:
[nit]: Do we usually mix camel and snake case?

Reply:
Haha good question. I have elsewhere, and Goland suggested it. It looks not very common in cockroachdb, but there are a handful of example, such as https://github.com/cockroachdb/cockroach/blob/3b2245f5e0fb5e17b4607f48dda5860dde841904/pkg/sql/opt/props/func_dep_test.go#L34